docs/api/proposed_endpoints.md

```diff
@@ -167,15 +167,15 @@ posted_time   | datetime    | time at which the post will be posted
 **Request**
 ```json
 {
-    "id": "0f342ac1-ac32-4bd1-3612-efa32bc3d9a0",
-    "posted_by": "01ade2ff-ab21-231f-a12b3c4d5e79",
+    "id": null,
```

Comment:
Remove all these.

Reply:
But this one is the POST request. Should I remove it even then also?